xml/crm.dtd

```diff
@@ -65,7 +65,7 @@ Additionally, num_peers and have_quorum are used during the election process to
  * ccm_transition stores the membership instance from the ConsensusClusterMembership layer.
  * cib_feature_revision is the feature set that this configuration requires
 -->
-<!ELEMENT configuration (crm_config, nodes, resources, constraints)>
+<!ELEMENT configuration (crm_config, nodes, resources, constraints, alerts)>
```

Comment:
Ah, that made me curious. It looks like RNG_cfg_base in xml/Makefile.am is the key.